crates/compiler/test_syntax/tests/snapshots/pass/suffixed_one_def.full.formatted.roc

```diff
@@ -0,0 +1,18 @@
+app "desugar-bang"
+    packages {
+        cli: "../basic-cli/platform/main.roc",
+    }
+    imports [
+        cli.Stdout,
+    ]
+    provides [main] to cli
+
+main =
+    # is this a valid statement?
+    "Foo" |> A.x!
+    # what about this?
+    "Bar"
+    |> B.y!
+        { config: "config" }
```

Comment:
I have been scratching my head over this. Why do we remove the newline preceding comments that are before statements? 

Also, it's not something that is changed in this PR, but I wasn't sure if we want to format the pipeline on a newline. I don't think this is an issue.

Reply:
We probably shouldn't remove it! I'm not sure why we would have been doing that either. 🤔